addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -6,6 +6,7 @@ const components = {
 };
 const {
     afterEach: utilsAfterEach,
+    afterNextRender,
```

Comment:
Impressive to see we had 500 LOCs of tests that didn't require this.
(and even around `this.component.mount()`, because we need it for some components that trigger several re-renders due to side-effects on mount)